src/templates/layouts/event/index.tsx

```diff
@@ -27,16 +28,40 @@ export const Event = ({
   link,
   additionalInfo,
   eventCTA,
+  emailCTA,
+  howToSignUp,
   body,
   listing,
 }: FormattedEvent) => {
   const [showRecurringEvents, setShowRecurringEvents] = useState(false)
+  const [mostRecentDate, setMostRecentDate] = useState(null)
+  const [showAllEvents, setShowAllEvents] = useState(false)
+  const initialFormattedDates = formatDateObject(datetimeRange).slice(0, 5)
+  const [currentFormattedDates, setCurrentFormattedDates] = useState(
+    initialFormattedDates
+  )
+  const formattedDates = formatDateObject(datetimeRange)
```

Comment:
can this go above `initialFormattedDates` and then `initialFormattedDates = formattedDates.slice(0.5)`

?

Reply:
I dont see why not, fixed